home/service/details.py

```diff
@@ -12,31 +12,25 @@ def __init__(self, urn: str):
         self.urn = urn
         self.client = self._get_catalogue_client()
 
-        filter_value = [MultiSelectFilter("urn", [urn])]
-        search_results = self.client.search(query="", page=None, filters=filter_value)
+        self.database_metadata = self.client.get_database_details(self.urn)
 
-        if not search_results.page_results:
+        if not self.database_metadata:
             raise ObjectDoesNotExist(urn)
 
-        self.result = search_results.page_results[0]
-
-        self.is_esda = any(
-            term.display_name == "Essential Shared Data Asset (ESDA)"
-            for term in self.result.glossary_terms
-        )
-
+        self.is_esda = any(term.display_name == "Essential Shared Data Asset (ESDA)"
+            for term in self.database_metadata.glossary_terms)
         self.entities_in_database = self._get_database_entities()
         self.context = self._get_context()
 
     def _get_database_entities(self):
         # we might want to implement pagination for database children
-        # details at some point
-        database_search = self.client.list_database_tables(
+        # details    at some point
```

Comment:
```suggestion
        # details at some point
```

don't think this diff was intentional